hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -528,8 +528,9 @@ public DeltaSyncService(Config cfg, JavaSparkContext jssc, FileSystem fs, Config
       this.jssc = jssc;
       this.sparkSession = SparkSession.builder().config(jssc.getConf()).getOrCreate();
       this.asyncCompactService = Option.empty();
+      this.props = properties.get();
 
-      if (fs.exists(new Path(cfg.targetBasePath))) {
+      if (fs.exists(new Path(cfg.targetBasePath)) && !Boolean.valueOf(props.getString("hoodie.deltastreamer.source.dfs.full.overwrite", "false"))) {
```

Comment:
since we have introduced a constant for this config, we'd better use the constant name. 
besides, provide a method to acquire it should be better